ETH-Merge-Bots/Changes-in-Block-Difficulty/src/utils.ts

```diff
@@ -0,0 +1,43 @@
+import { Finding, FindingSeverity, FindingType } from "forta-agent";
+import BigNumber from "bignumber.js";
+
+export const NUMBER_OF_BLOCKS_TO_CHECK: number = 100;
+export const FINDING_THRESHOLD: BigNumber = new BigNumber(1.5);
+
+export const TERMINAL_TOTAL_DIFFICULTY: BigNumber = new BigNumber("58750000000000000000000");
+
+export const createFinding = (
+  blockDifficulty: string,
+  movingAverage: string,
+  changePercentage: string,
+  threshold: string
+): Finding => {
+  if (blockDifficulty > movingAverage) {
+    return Finding.fromObject({
+      name: "Unusual Block Difficulty Increase Detection",
+      description: `Block difficulty increased more than ${threshold}% compared to the moving average`,
```

Comment:
I think we could add the amount of blocks the moving average covers to the description. Thoughts?